challenge-02/README.md

```diff
@@ -35,19 +42,26 @@ Crie uma função com as seguintes características:
     Preencha todos os valores corretamente!
 3. O retorno da função deve ser a multiplicação dos 3 argumentos, somando `2` ao resultado da multiplicação.
 */
-?
+
+function recebeargumentos( a , b , c ) {
+	if( a && b && c === undefined ) {
```

Comment:
-1
Está funcionando, mas não da forma correta.Tente invocar a função somente com um parâmetro pra você ver o problema :)

O problema aqui é o seguinte: 
provavelmente sua intenção era verificar se `a`, `b` e `c` seriam igual a `undefined`, correto?

Mas nesse caso, você está verificando somente se `c` é igual a `undefined`.

A cada vez que você usa o `&&`, o Javascript faz uma nova avaliação do código. Então o seu código está dizendo o seguinte:

> Se o `a` estiver definido com algum valor, E o `b` também estiver definido com algum valor, MAS o `c` for igual a `undefined`, então...

Ou seja: quando você testa somente com o argumento no `if`, sem usar o sinal de `===`, o Javascript pensa que você está perguntando o seguinte:

> Esse argumento foi passado com algum valor?

Se algum valor for passado, ele vai retornar `true` para essa verificação. Caso contrário, vai retornar `false`. Você vai entender melhor essa parte quando assistir a aula `4`, que fala sobre `truthy` e `falsy` :)

Mas basicamente, para arrumar isso, você se precisa testar se QUALQUER UM DESSES ARGUMENTOS não esteja definido. Da forma que você está fazendo, você precisa que todas as avaliações sejam verdadeiras, mas o enunciado pede para que QUALQUER UMA que não seja, ele já mostre a mensagem que `todos os valores devem ser preenchidos`.

Conseguiu compreender? Se ficou dúvidas com relação a isso, me avise :)